.gitignore

```diff
@@ -21,4 +21,4 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 public/index.html~HEAD
-package-lock.json
```

Comment:
Why did you remove the package-lock.json from .gitignore? This file changes for almost everyone and people will have to deal with many merge conflicts and a large diff file like [below](https://github.com/openmrs/openmrs-ocl-client/pull/2/files#diff-32607347f8126e6534ebc7ebaec4853d)